README.md

```diff
@@ -110,6 +110,13 @@ sudo git clone https://github.com/ahmetb/kubectx /opt/kubectx
 sudo ln -s /opt/kubectx/kubectx /usr/local/bin/kubectx
 sudo ln -s /opt/kubectx/kubens /usr/local/bin/kubens
 ```
+
```

Comment:
Do you mind doing this:

```
### Debian/Ubuntu

....

### Other Linux

(the instructions above this line basically)
```

Reply:
I'd add "Other Linux" line after we'll have enough content for the distro sections, but added the explicit section mentioning Debian/Ubuntu.

Reply:
I mostly said that because in terms of Desktop usage it probably goes like:

- Debian/Ubuntu
- Arch Linux 
- Other Linux 

(not sure about the last two but it feels like Arch should appear first, then other as a "catch all"). But this is fine as is.